UI/DevScreens.cpp

```diff
@@ -1105,3 +1107,90 @@ void ShaderViewScreen::CreateViews() {
 
 	layout->Add(new Button(di->T("Back")))->OnClick.Handle<UIScreen>(this, &UIScreen::OnBack);
 }
+
+const std::string framedumpsBaseUrl = "http://framedump.ppsspp.org/repro/";
+
+FrameDumpTestScreen::FrameDumpTestScreen() {
+
+}
+
+FrameDumpTestScreen::~FrameDumpTestScreen() {
+	g_DownloadManager.CancelAll();
+}
+
+void FrameDumpTestScreen::CreateViews() {
+	using namespace UI;
+
+	root_ = new AnchorLayout(new LayoutParams(FILL_PARENT, FILL_PARENT));
+	auto di = GetI18NCategory("Dialog");
+
+	TabHolder *tabHolder;
+	tabHolder = new TabHolder(ORIENT_VERTICAL, 200, new AnchorLayoutParams(10, 0, 10, 0, false));
+	root_->Add(tabHolder);
+	AddStandardBack(root_);
+	tabHolder->SetTag("DumpTypes");
+	root_->SetDefaultFocusView(tabHolder);
+
+	ViewGroup *dumpsScroll = new ScrollView(ORIENT_VERTICAL, new LinearLayoutParams(FILL_PARENT, FILL_PARENT));
+	dumpsScroll->SetTag("GameSettingsGraphics");
+	LinearLayout *dumps = new LinearLayout(ORIENT_VERTICAL);
+	dumps->SetSpacing(0);
+	dumpsScroll->Add(dumps);
+	tabHolder->AddTab("Dumps", dumps);
+
+	dumps->Add(new ItemHeader("GE Frame Dumps"));
+
+	for (auto &file : files_) {
+		std::string url = framedumpsBaseUrl + file;
+		Choice *c = dumps->Add(new Choice(file));
+		c->SetTag(url);
+		c->OnClick.Handle<FrameDumpTestScreen>(this, &FrameDumpTestScreen::OnLoadDump);
+	}
+}
+
+UI::EventReturn FrameDumpTestScreen::OnLoadDump(UI::EventParams &params) {
+	std::string url = params.v->Tag();
+	INFO_LOG(COMMON, "Trying to launch '%s'", url.c_str());
+	// Our disc streaming functionality detects the URL and takes over and handles loading framedumps well,
+	// except for some reason the game ID.
```

Comment:
It should embed the game ID in dumps now and use that - see #13396.  This is only in newer dump files.

-[Unknown]

Reply:
I'm aware of that, but I took fresh dumps, and looked at the logs when loading them through http this way. They do not manage to load the game ID. Have not yet debugged it.

Reply:
Hm, for me it reaches `coreParameter.compat.Load(discID);` with the value "ULUS10336".  It still logs generating a fake one because that logic never affected the SFO code.

-[Unknown]

Reply:
Oh right, I guess the logs are a bit misleading then, and thus, this comment is too :)

I didn't look into it carefully, just noted that we generated an ID...